src/components/tag-input/tag-input.component.html

```diff
@@ -4,7 +4,10 @@
     [class.ux-tag-input-icon-inset]="icon"
     (click)="toggle()">
 
-    <li *ngFor="let tag of tags; let i = index" class="ux-tag"
+    <li uxFocusIndicator
+        mouseFocusIndicator="true"
+        programmaticFocusIndicator="true"
+        *ngFor="let tag of tags; let i = index" class="ux-tag"
```

Comment:
Just a slight formatting suggestion, but here every attribute is on a separate line, with the exception of `class`, could it be moved to its own line to be consistent.